docs/Guidelines.md

```diff
@@ -0,0 +1,78 @@
+# Overview
+The purpose of *this* document is to serve as a primer for developers/qe who are looking for best practices or adding new tests to RHTAP E2E framework.
+
+## General tips
+
+* Make sure you've implemented any required controller functionality that is required for your tests within the following files
+   * `pkg/utils/<new controller directory>` - net new controller logic for a new service or component
+   * `pkg/framework/framework.go` - import the new controller and update the `Framework` struct to be able to initialize the new controller
+* Every test package should be imported to `cmd/e2e_test.go`, e.g. [has](https://github.com/redhat-appstudio/e2e-tests/blob/main/cmd/e2e_test.go#L15).
```

Comment:
```suggestion
* Every test package should be imported to `cmd/e2e_test.go`, e.g. [byoc](https://github.com/redhat-appstudio/e2e-tests/blob/main/cmd/e2e_test.go#L15).
```

Reply:
imo there should be a permanent link, so it doesn't change in a future, i.e. https://github.com/redhat-appstudio/e2e-tests/blob/b3f6894dd039533a34385092ba123b3e3d940d98/cmd/e2e_test.go#L15